string pose_id
---
bool status
